src/Sylius/Behat/Context/Ui/Admin/ManagingZonesContext.php

```diff
@@ -356,14 +356,14 @@ private function assertZoneAndItsMember(ZoneInterface $zone, ZoneMemberInterface
     }
 
     /**
-     * @Then /^I can not add a(?: country| province| zone) "([^"]+)"$/
+     * @Then /^I can not add a ([^"]+) "([^"]+)"$/
```

Comment:
```suggestion
     * @Then /^I cannot add a ([^"]+) "([^"]+)"$/
```
though 
```suggestion
     * @Then /^I should not be able to add a ([^"]+) "([^"]+)"$/
```
would be better